ip_country/src/test_dbip_country.rs

```diff
@@ -0,0 +1,22 @@
+
+// GENERATED CODE: REGENERATE, DO NOT MODIFY!
```

Comment:
Is this really also automatically generated file? Doesn't the comment lie?

Reply:
Comment does not lie. This is manually generated by ip_country binary, to establish set of ips for tests.

Reply:
Manually generated? 

I think this definitely isn't generated by any binary as the other official file is. So the comment is out of place here. While it was written for the original file (the prod code), it makes no sense to have it here.